### PR TITLE
[#11] Add FOISA help page

### PR DESCRIPTION
**Pull request by HelenWDTK**
## Relevant issue(s)
#11 
## What does this do?
Adds a page about the Freedom of Information Scotland Act
## Why was this needed?
We don't have guidance about Scotland on the Site
## Screenshots
Screenshot 2023-06-28 at 08 13 22

Screenshot 2023-06-28 at 08 13 44

### CODE REVIEW

File: lib/views/help/freedom_of_information_scotland_act.html.erb [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+  You can contact the Scottish Information Commissioner by email on enquiries@itspublicknowledge.info or by post:
+  </p>
+  <p>
+  Scottish Information Commissioner<br>
+  Kinburn Castle<br>
+  Doubledykes Road<br>
+  St Andrews<br>
+  Fife<br>
+  KY16 9DS<br>
+  </p>
```
Comment (mdeuk) [inline lib/views/help/freedom_of_information_scotland_act.html.erb:154]:
```suggestion
  <p>
  You can contact the Scottish Information Commissioner by email at: <b><a href="mailto:enquiries@itspublicknowledge.info">enquiries@itspublicknowledge.info</a></b>, or by post:
  </p>
  <p>
  Scottish Information Commissioner<br>
  Kinburn Castle<br>
  Doubledykes Road<br>
  ST ANDREWS<br>
  KY16 9DS<br>
  </p>
  <p>
  You can also contact them for advice, by calling their helpline on <b>01334 464 610</b>, or if you use <abbr title="British Sign Language">BSL</abbr>, by using <a href="http://contactscotland-bsl.org">Contact Scotland BSL</a>.
  </p>
```

I thought it'd be helpful to call out the helpline here [^1].

Pedantry: Royal Mail [confirms](https://www.royalmail.com/find-a-postcode) the post town is St Andrews, so we can reduce some of the address by stopping there. 🙂 [^2]

[^1]: The BSL link here is the Scottish Government funded VRS, which OSIC are required to promote (as per their [BSL plan](https://www.itspublicknowledge.info/strategic-and-operational-plans). It's important that we call this out.
[^2]: Technically, they say "ST. ANDREWS", but the full stop is probably superfluous.

File: lib/config/wdtk-routes.rb [hunk 2]
```diff
@@ -38,7 +41,22 @@
...
+  get '/help/exemptions' => 'help#exemptions',
+      as: :help_exemptions
+
+  get '/help/authority_performance_tracking' => 'help#authority_performance_tracking',
+      as: :help_authority_performance_tracking
+
+  get '/help/search_engines' => 'help#search_engines',
+      as: :help_search_engines
+  
+   get '/help/freedom_of_information_scotland_act' => 'help#freedom_of_information_scotland_act',
```
Comment (mdeuk) [inline lib/config/wdtk-routes.rb:60]: I'm not a fan of the long URL. Can we alias this, so a shorter version is also available?

**Review by mdeuk — CHANGES_REQUESTED** [review]
This is looking great. 👍 

I've added some suggestions, which are hopefully not too controversial.

File: lib/views/help/_sidebar.html.erb [hunk 3]
```diff
@@ -54,6 +54,15 @@
...
         <%= link_to_unless_current "Environmental Information", help_environmental_information_path %>
       </li>
+      <li>
+        <%= link_to_unless_current "Accessing Information", help_accessing_information_path %>
+      </li>
+      <li>
+        <%= link_to_unless_current "Exemptions", help_exemptions_path %>
+      </li>
+      <li>
+        <%= link_to_unless_current "Freedom of Information (Scotland) Act", help_freedom_of_information_scotland_act_path %>
```
[the remark this answers is not in the document]
Comment (HelenWDTK) [reply lib/views/help/_sidebar.html.erb:64]: This makes sense, but I also think this needs to be consistent with whatever is decided for the FOIA page, so we need to pick

File: lib/config/wdtk-routes.rb (hunk 2 above)
Comment (HelenWDTK) [reply lib/config/wdtk-routes.rb:60]: This was to keep it consistent with the new page at https://github.com/mysociety/whatdotheyknow-theme/pull/1728. If we change this, that will need modifying too.

File: lib/views/help/freedom_of_information_scotland_act.html.erb [hunk 4]
```diff
@@ -0,0 +1,160 @@
+<% @title = "Freedom of Information (Scotland) Act" %>
+<%= render :partial => 'sidebar' %>
+<div id="left_column_flip" class="left_column_flip">
+  <h1 id="freedom_of_information_scotland_act"><%= @title %></h1>
+
+  <h2 id="what_is_FOISA">
+    What is the Freedom of Information (Scotland) Act 2002?
+    <a href="#what_is_FOISA">#</a>
```
[the remark this answers is not in the document]
Comment (HelenWDTK) [reply lib/views/help/freedom_of_information_scotland_act.html.erb:8]: Not that I could find. I think it does need to be the name of the legislation really.

File: lib/views/help/_sidebar.html.erb (hunk 3 above)
[the remark this answers is not in the document]
Comment (mdeuk) [reply lib/views/help/_sidebar.html.erb:64]: > This makes sense, but I also think this needs to be consistent with whatever is decided for the FOIA page, so we need to pick

We could have `About FOI` / `About FOI in Scotland`, or better yet: `About FOI` with links off to both?

[the remark this answers is not in the document]
Comment (HelenWDTK) [reply lib/views/help/_sidebar.html.erb:64]: For now, I'm going to go with About FOI in Scotland and About FOI, as that fits on one line

File: lib/views/help/freedom_of_information_scotland_act.html.erb [hunk 5]
```diff
@@ -0,0 +1,165 @@
+<% @title = "Freedom of Information (Scotland) Act" %>
+<%= render :partial => 'sidebar' %>
+<div id="left_column_flip" class="left_column_flip">
+  <h1 id="freedom_of_information_scotland_act"><%= @title %></h1>
+
+  <h2 id="what_is_FOISA">
+    What is the Freedom of Information (Scotland) Act 2002?
```
Comment (gbp) [inline lib/views/help/freedom_of_information_scotland_act.html.erb:7]:
```suggestion
    What&nbsp;is&nbsp;the&nbsp;Freedom&nbsp;of&nbsp;Information&nbsp;(Scotland)&nbsp;Act&nbsp;2002?
```

File: lib/views/help/freedom_of_information_scotland_act.html.erb (hunk 4 above)
[the remark this answers is not in the document]
Comment (gbp) [reply lib/views/help/freedom_of_information_scotland_act.html.erb:8]: Added a suggestion which should stop this from wrapping.

File: lib/config/wdtk-routes.rb (hunk 2 above)
Comment (HelenWDTK) [reply lib/config/wdtk-routes.rb:60]: The people have spoken and picked short urls 🎉  I'll fix these across the two pages when I get a sec.

**Review by mdeuk — APPROVED** [review]
This looks brilliant, many thanks for putting it together - and wrangling all the changes @HelenWDTK! 👍

**Review by garethrees — APPROVED** [review]
Fixed a minor routing issue and squashed the commits. Good to go 🏴󠁧󠁢󠁳󠁣󠁴󠁿

**Pull request merged**